go/engine/login_test.go

```diff
@@ -611,15 +609,36 @@ func TestProvisionAutoreset(t *testing.T) {
 	// Travel 3 days into future + 1h to make sure that it all runs
 	require.NoError(t, accelerateReset(tcX))
 	require.NoError(t, timeTravelReset(tcX, time.Hour*73))
-	time.Sleep(1500 * time.Millisecond)
+
+	// Rather than sleeping we'll wait for autoreset by analyzing its state
```

Comment:
could simplify this by dropping `found` and having `require.NoError(t, lastErr)` at the end of the loop